### ISSUE TITLE
[ISO]: introduce neon-wayland ISO

### ISSUE THREAD

**Issue by star-buck** [blue-systems/plasma-5.4#154]
in spirit of rebecca black and unity8-mir-desktop previews, provide a weekly ISO for plasma-desktop-wayland

**Comment by apachelogger** [comment]
Is blocked until @shadeslayer fixes the tooling so we can do deployments again. Until then I have no means to deploy new code to build a different ISO.

Also @mgraesslin would need to put a wayland-session file somewhere for the DM to use so one can login.

Also @davidedmundson would need to make sure sddm can actually use the wayland session file (or get people to do wayland support ;))

Once all that is done we can start doing ISOs and probably do some packaging adjustment to make the ISO hand over to wayland.

**Comment by mgraesslin** [comment]
I have no idea about what is needed for a wayland-session file for the DM. I hope someone else can pick up that task.

Personally I disagree that we need such an iso. I think that's exactly what rebecca black OS is for. We got very good feedback from rebecca black OS developer and I don't want to destroy that relationship by providing a competitive product.

**Comment by star-buck** [comment]
By that logic we should stop working on anything neon, because thats a competing product with all other distros and destroy our good relationship with anyone else.

**Comment by mgraesslin** [comment]
No, the situation is different. While there are many distros and neon is just one of many, there is only one "test Wayland" distro.

**Comment by bhush9** [comment]
We are not really providing competitive product. I don't think providing neon-wayland iso would count as competitive or will destroy good relationship.

**Comment by star-buck** [comment]
still, the argument of "not competing" is the same, be it then 2 distros or 102.

**Comment by mgraesslin** [comment]
well to be honest: I talked with many distro developers during Akademy and later and they feel threatened by Neon. So yes Neon is dangerous in kind of the same way. But that gets offtopic.

My point is actually more the first part: rebacca black os is exactly what is needed. I don't think (at the current state) such an iso provided by us makes sense or would help us during development. As long as important functionality (like e.g. move/resize windows) are not working it just doesn't make sense to put time on it (that's why I did not spent any time on thinking about session support for the 5.4 release: it's too early). Once we get to a state that the whole thing becomes dogfoodable we should start looking into a session file and then an iso.

Given the current state it might be better to extend our existing ISO to easily test Wayland: ship the x11 backend, provide a desktop file to run a nested plasma/wayland session on the live iso. Maybe also an easy way to start a full (DRM) plasma/wayland session on the live iso.
